pxtcompiler/emitter/service.ts

```diff
@@ -1107,6 +1107,11 @@ namespace ts.pxtc.service {
                 }
             }
 
+            if (resultSymbols.length === 0) {
```

Comment:
what does this mean, if the user enters garbage it receives a ton of suggestions?

Reply:
It means that: "if by this point we don't yet have a specialized set of results (like those for member completion or a specific type for a call expression), use all symbols as the start"
So yes, this results in a flood of symbols but monaco filters and searches these down based on what you're typing. This is how vs code works. Type "fo" and it'll show you all symbols with "f" and "o" in them sorted appropriately.

Reply:
I updated the comment to clarify